### ISSUE TITLE
ddev start with webcache_enabled throws error in linux

### ISSUE THREAD

**Issue by mistermoper** [drud/ddev#1424]
**Describe the bug**
When I start a ddev project with webcache_enabled, the following error is displayed in the console:

`Unable to get project(s): webcache_enabled: true is not yet avalable for linux `.

Then the project does not starts and I cannot work with it.

When I set webcache_enabled as false it works correctly, but then mac os / windows users must modify the config.yaml file to use it . This file is commited in git so changing it can make we commit constantly this changes.

Would it be good idea to don't throw this error , and don't apply this feature in linux systems even the variable is set, so we could use the same config.yaml independently our O.S.? Maybe just a notice is good enough.

**To Reproduce**

1.  Have a ddev configuration with webcache_enabled in linux.
2. Make ddev start

**Expected behavior**
The ddev instance is up and I can use my site.

**Screenshots**
![ddev-project-webcache-enabled](https://user-images.githubusercontent.com/3202817/52113927-fac12080-260a-11e9-8e88-3ee112c34d77.jpg)

**Version and configuration information (please complete the following information):**
 - Host computer OS and Version: Ubuntu 17.10
 - Docker version information:
```
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:25:03 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:23:27 2018
  OS/Arch:          linux/amd64
  Experimental:     false
```

 - ddev version information:
```
ddev version
bgsync          drud/ddev-bgsync:v1.5.2       
cli             v1.5.2                        
commit          v1.5.2                        
db              drud/ddev-dbserver:v1.5.2-10.2
dba             drud/phpmyadmin:v1.5.2        
ddev-ssh-agent  drud/ddev-ssh-agent:v1.5.2    
docker          18.06.1-ce                    
docker-compose  1.22.0                        
domain          ddev.local                    
router          drud/ddev-router:v1.5.2       
web             drud/ddev-webserver:v1.5.2
```

 - config.yaml contents for the misbehaving project:
```
APIVersion: v1.5.2
name: ddevproject
type: drupal8
docroot: web
php_version: "7.1"
webcache_enabled: true
webserver_type: nginx-fpm
router_http_port: "80"
router_https_port: "443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
mariadb_version: "10.2"
webcache_enabled: true
provider: default


# This config.yaml was created with ddev version v1.5.1 
# webimage: drud/ddev-webserver:v1.5.1
# dbimage: drud/ddev-dbserver:v1.5.0-10.2
# dbaimage: drud/phpmyadmin:v1.5.0
# bgsyncimage: drud/ddev-bgsync:v1.5.0
# However we do not recommend explicitly wiring these images into the
# config.yaml as they may break future versions of ddev.
# You can update this config.yaml using 'ddev config'.

# Key features of ddev's config.yaml:

# name: <projectname> # Name of the project, automatically provides
#   http://projectname.ddev.local and https://projectname.ddev.local

# type: <projecttype>  # drupal6/7/8, backdrop, typo3, wordpress, php

# docroot: <relative_path> # Relative path to the directory containing index.php.

# php_version: "7.1"  # PHP version to use, "5.6", "7.0", "7.1", "7.2", "7.3"

# You can explicitly specify the webimage, dbimage, dbaimage lines but this
# is not recommended, as the images are often closely tied to ddev's' behavior,
# so this can break upgrades.

# webimage: <docker_image>  # nginx/php docker image.
# dbimage: <docker_image>  # mariadb docker image.
# dbaimage: <docker_image>
# bgsyncimage: <docker_image>

# router_http_port: <port>  # Port to be used for http (defaults to port 80)
# router_https_port: <port> # Port for https (defaults to 443)

# xdebug_enabled: false  # Set to true to enable xdebug and "ddev start" or "ddev restart"

xdebug_enabled: true

# webserver_type: nginx-fpm  # Can be set to apache-fpm or apache-cgi as well

# additional_hostnames:
#  - somename
#  - someothername
# would provide http and https URLs for "somename.ddev.local"
# and "someothername.ddev.local".

# additional_fqdns:
#  - example.com
#  - sub1.example.com
# would provide http and https URLs for "example.com" and "sub1.example.com"
# Please take care with this because it can cause great confusion.

# upload_dir: custom/upload/dir
# would set the destination path for ddev import-files to custom/upload/dir.

# working_dir:
#   web: /var/www/html
#   db: /home
# would set the default working directory for the web and db services. 
# These values specify the destination directory for ddev ssh and the 
# directory in which commands passed into ddev exec are run. 

# omit_containers: ["dba", "ddev-ssh-agent"]
# would omit the dba (phpMyAdmin) and ddev-ssh-agent containers. Currently
# only those two containers can be omitted here.
# Note that these containers can also be omitted globally in the 
# ~/.ddev/global_config.yaml or with the "ddev config global" command.


# provider: default # Currently either "default" or "pantheon"
#
# Many ddev commands can be extended to run tasks after the ddev command is
# executed.
# See https://ddev.readthedocs.io/en/stable/users/extending-commands/ for more
# information on the commands that can be extended and the tasks you can define
# for them. Example:
#hooks:
# post-import-db:
#   - exec: drush cr
#   - exec: drush updb
```
 - Do you have any custom configuration (nginx, php, mysql) in the .ddev folder? No

**Comment by rfay** [comment]
This is a duplicate of #1395 right? It's fixed in #1390 (next release, about 2 weeks). You can also set `developer_mode: true` in ~/.ddev/global_config.yaml and webcache_enabled can be turned on in linux.

**Comment by rfay** [comment]
This was fixed in #1390, closing.
